src/core/thread/mesh_forwarder.cpp

```diff
@@ -1573,21 +1573,21 @@ const char *MeshForwarder::MessagePriorityToString(const Message &aMessage)
 
     switch (aMessage.GetPriority())
     {
+    case Message::kPriorityNetworkControl:
+        priorityText = "networkcontrol";
```

Comment:
How about "net" instead of "networkcontrol"?
Suggesting this since this is going to be added on IPv6 message logs and shorter would be better.

Reply:
Will update it.